src/Glpi/Form/QuestionType/AbstractQuestionTypeShortAnswer.php

```diff
@@ -103,6 +113,7 @@ class="form-control"
                 placeholder="{{ input_placeholder }}"
                 value="{{ question is not null ? question.fields.default_value : '' }}"
                 aria-label="{{ __('Default value') }}"
+                {{ attributes|map((value, key) => '%s=%s'|format(key, value))|join(' ') }}
```

Comment:
This works only if you have no spacing chars in your attribute value.

Something like this would be preferable (to be tested with spacing chars/quotes/...
```suggestion
                {{ attributes|map((value, key) => '%s="%s"'|format(key|e, value|e('html_attr')))|join(' ')|raw }}
```

Reply:
To be honest, a simple for loop would also work and be way more readable.